shared/js/template.js

```diff
@@ -114,24 +114,25 @@
    *                          HTML that is known and are
    *                          "known" to ignore.
    */
-  Template.prototype.interpolate = function(data, options) {
-    // This _should_ be rewritten to use Firefox's support for ES6
-    // default parameters:
-    // ... = function(data, options = { safe: [] }) {
-    //
-    options = options || {};
-    options.safe = options.safe || [];
```

Comment:
We still need this in case an `options` object is given without a `safe` property. So it's better to just have `{}` as default value for `options` and keep the second line here.